### PR TITLE
Added more courses to Computer Networking

### PR DESCRIPTION
**Pull request by PuneetTri**
## What does this PR do?
Adding more courses for computer networks, in-depth and great for beginners.

### Why is this valuable (or not)?
Both are in-depth courses on Computer Networking, but starting off from basics.

### How do we know it's really free?
It's on youtube!

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -989,6 +989,8 @@
 
 ### Networking
 
+* [Computer Networking](https://www.youtube.com/watch?v=IPvYjXCsTg8&t=3520s) - Kunal Kushwaha
```
Comment (shrn01) [inline courses/free-courses-en.md:992]:
```suggestion
* [Computer Networking](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
```
Remove the timestamp

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -989,6 +989,8 @@
 
 ### Networking
 
+* [Computer Networking](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
```
Comment (shrn01) [inline courses/free-courses-en.md:992]:
```suggestion
* [Computer Networking Full Course](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
```
Use the same name given by the resource

Comment (PuneetTri) [reply courses/free-courses-en.md:992]: Does it really matter? Either may I'm committing your suggestion. And, please suggest all the suggestion in one go

**Comment by shrn01** [comment]
Fix the Alphabetical sorting of the resources

File: courses/free-courses-en.md (hunk 2 above)
Comment (shrn01) [reply courses/free-courses-en.md]: Hmm yeah, it's in the contribution guidelines. Many resources have license to share but with proper attribution, so we prefer to have the same name that is in the resource

**Review by shrn01 — APPROVED** [review]

**Pull request merged**